### DISCUSSION TITLE
Week 3

### DISCUSSION THREAD

**Discussion by ColinFay** [ColinFay/ur-first-5k#110] [General]
Hey y'all! 

Welcome to week 3 :) 

Happy to see most of you have be following the program!

Here is this week menu: 

```
── Day 1 ─────────
→ W03D1-Cross Training
Cross train, 30 minutes. Suggestions can be found at: https://github.com/ColinFay/ur-first-5k/discussions/109

── Day 2 ─────────
→ W03D2-Recovery Run
• Run, easy pace, 15 minutes.
• Cool down, 5 to 10 minutes.
• Stretch.

── Day 3 ─────────
→ Rest
Rest day.

── Day 4 ─────────
→ W03D4-Easy Run
• Run, easy pace, 10 minutes.
• Recovery walk, 5 minutes.
• Run, easy pace, 10 minutes.
• Cool down, 5 to 10 minutes.
• Stretch

── Day 5 ─────────
→ Rest
Rest day.

── Day 6 ─────────
→ Rest
Optional: Swap Sunday's run to today.

── Day 7 ─────────
→ W03D7-Easy Run
• Run, easy pace, 15 minutes.
• Cool down, 5 to 10 minutes.
• Stretch.
```

Feel free to use this discussion thread if you have any question, suggestion, or if you just want to share your experience :) 

Enjoy ! 

Colin

**Comment by MaelaKloareg** [comment]
Well I must admit it’s easier than expected ! It’s going pretty smoothly, although a bit wet and muddy today 😅

**Reply by ColinFay** [reply]
So glad to hear that :) 

Happy to see that this is going well. Let's hope this new week will have more ☀️
